### PR TITLE
New alt-geom for SF locality

### PR DESCRIPTION
**Pull request by stepps00**
Related to #316.

This PR includes two updates:

- Renames the existing `85922583-alt-mapzen.geojson` file to `85922583-alt-mapzen-land.geojson`
- Adds a new `85922583-alt-mapzen.geojson` alt_geometry file.

`85922583-alt-mapzen-land.geojson`: Just the portion of San Francisco on the Peninsula.
`85922583-alt-mapzen.geojson`: The portion of San Francisco on the Peninsula, plus Treasure Island.

**References:** https://github.com/whosonfirst/whosonfirst-cookbook/blob/master/how_to/creating_alt_geometries.md

### CODE REVIEW

File: data/859/225/83/85922583-alt-mapzen.geojson [hunk 1]
```diff
@@ -1,16 +1,7 @@
...
-}
\ No newline at end of file
+"type": "FeatureCollection",
+"crs": { "type": "name", "properties": { "name": "urn:ogc:def:crs:OGC:1.3:CRS84" } },
```
Comment (nvkelso) [inline data/859/225/83/85922583-alt-mapzen.geojson]: This CRS entry is surprising.

File: data/859/225/83/85922583-alt-mapzen.geojson [hunk 2]
```diff
@@ -1,16 +1,7 @@
...
-}
\ No newline at end of file
+"type": "FeatureCollection",
```
Comment (nvkelso) [inline data/859/225/83/85922583-alt-mapzen.geojson]: Shouldn't be a feature collection.

File: data/859/225/83/85922583-alt-mapzen.geojson [hunk 3]
```diff
@@ -1,16 +1,7 @@
...
...01160663000076 ], [ -122.397632493999936, 37.799858370000038 ], [ -122.396741187999908, 37.798834498000076 ], [ -122.396588103999932, 37.798842932000071 ], [ -122.395581486999902, 37.799459439000088 ], [ -122.394436858999939, 37.800181518000045 ], [ -122.394338331999904, 37.800157742000067 ], [ -122.394252250999898, 37.800073542000064 ], [ -122.394244236999896, 37.799994427000058 ], [ -122.395460759999935, 37.799348853000083 ], [ -122.396512746999917, 37.798764128000073 ], [ -122.39651899799992, 37.79873281600004 ], [ -122.396529431999909, 37.798675234000086 ], [ -122.395861034999939, 37.797871101000055 ], [ -122.39382566799992, 37.798942350000061 ], [ -122.393554264999921, 37.798641781000072 ], [ -122.395455150999908, 37.79754869900006 ], [ -122.395493442999907, 37.797469730000046 ], [ -122.39542279099993, 37.79739253300005 ], [ -122.395304461999899, 37.797394435000058 ], [ -122.395131680999896, 37.797494934000042 ], [ -122.39499621199991, 37.79734122900004 ], [ -122.395176716999913, 37.79723623700005 ], [ -122.395203960999936, 37.797153427000069 ], [ -122.395174586999929, 37.797121331000085 ], [ -122.394876734999912, 37.79725845300004 ], [ -122.393210063999902, 37.798248977000071
```
Comment (nvkelso) [inline data/859/225/83/85922583-alt-mapzen.geojson]: Has this been exportified? There are some extra properties here which are surprising.

**Comment by stepps00** [comment]
Closing this issue. Tracking this update through #747.

**Pull request closed without merging**